flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
I thought luajit can work with aarch64 too?

Reply:
yes it does

Reply:
This is used in companion with the Windows game Path of Exile (which runs very will through Proton, hence the interest) which is x86-64 only. Also, the C++ pobfrontend implementation has never been tested on aarch64 either AFAIK, so I'm hesitant...

Reply:
Please try removing this after merge.